editor/src/clj/editor/protobuf_forms_util.clj

```diff
@@ -0,0 +1,8 @@
+(ns editor.protobuf-forms-util
+  (:require [dynamo.graph :as g]))
```

Comment:
* Add copyright header.
* Add `(set! *warn-on-reflection* true)`, and make sure to fix any reflection warnings.